roles/kubernetes-control-plane/tasks/main.yml

```
@@ -1,3 +1,175 @@
+- name: Pre configure control plane hooks
+  ansible.builtin.include_tasks: "{{ item }}"
+  with_items: "{{ kubernetes_hookfiles.pre_configure_control_planes }}"
+  when:
+  - kubernetes_hookfiles.pre_configure_control_planes is defined
+
+- name: Create kubernetes/config/patches directory
+  ansible.builtin.file:
+    dest: "{{ item }}"
+    state: directory
+    mode: '0700'
+  with_items:
+  - "{{ kubernetes_config_directory }}/patches"
+  - "{{ kubernetes_scripts_directory }}"
```

Change order to create them recursively.
Change item to something unique to avoid warning/duplicating use of item.
```suggestion
- name: Create kubernetes/config/patches directory
  loop:
  - "{{ kubernetes_scripts_directory }}"
  - "{{ kubernetes_config_directory }}/patches"
  loop_control:
    loop_var: patch_directory
  ansible.builtin.file:
    dest: "{{ patch_directory }}"
    state: directory
    mode: '0700'
```